public/app/features/plugins/admin/components/InstallControls/ExternallyManagedButton.tsx

```diff
@@ -35,7 +37,12 @@ export function ExternallyManagedButton({ pluginId, pluginStatus }: ExternallyMa
   }
 
   return (
-    <LinkButton href={externalManageLink} target="_blank" rel="noopener noreferrer">
+    <LinkButton
+      disabled={!config.angularSupportEnabled && angularDetected}
```

Comment:
TODO: What do we do if the plugin needs to be updated? ("Update via grafana.com" some lines above).

I'm not sure if we can enable/disable depending on whether the target update version is using Angular or not

Reply:
The same also applies when plugins aren't managed externally (InstallControlButtons.tsx)

Reply:
is there a way to permit updates, but disable new installs?

Reply:
Good question, maybe reach out to someone in @grafana/hosted-grafana-team for some input?

Reply:
I tested the various scenarios and with the current code (always allow update, allow installing only if angular is enabled) and it works as expected.

It's always possible to update plugins when Angular is enabled.
When Angular is disabled, it's possible to "update" (or more correctly, re-install) an old Angular plugin only if it has been updated to React.

I am marking this as resolved.